modules/custom_operations/user_ie_extensions/tokenizer/utils.cpp

```diff
@@ -90,7 +90,7 @@ void set_ragged_output(Node* node, size_t output_index, const PartialShape& shap
 }
 
```

Comment:
should we also remove https://github.com/openvinotoolkit/openvino_contrib/blob/352a4e3ff9bac77a68062371faa5c1b7c1c2bbac/modules/custom_operations/user_ie_extensions/include/openvino_extensions/strings.hpp and similar things in python code?

Reply:
Finally -- yes. I would like to avoid it now because we still support this legacy format as an input and OVMS adopted this format for MUSE model with sentence piece tokenizer. When all dependent components will use string tensors we can remove it. As an alternative, or the next step, we can move these functions to OVMS side.

@dtrawins, do you think we can do that in this release? Do we know about native users of MUSE model besides OVMS? Do we have other extensions based on that u8 packed tensor in OVMS besides ops in openvino_contrib?

Reply:
old users can stick to old commits, while in master we can drop all these things.

Reply:
@dtrawins, is this a requirement from OVMS still support u8 packed string format instead of supporting native strings? As this is contrib repo we allow breaking changes and we would like to make it as clean as possible.